### ISSUE TITLE
[Backport] Audit etcd backup logs

### ISSUE THREAD

**Issue by Jono-SUSE-Rancher** [rancher/rancher#32826]
Just as was done in https://github.com/rancher/rancher/issues/32794 we need to make sure to backport this into the 2.5 line so we can prevent the log build-up on any of our users running large RKE clusters.

**Comment by Jono-SUSE-Rancher** [comment]
This needs to be validated on a v.2.5.9-RC as well as a v2.5-head.

**Comment by dramich** [comment]
https://github.com/rancher/rancher/pull/32870

**Comment by jiaqiluo** [comment]
available for testing in rancher `v2.5.9-rc1`

**Comment by anupama2501** [comment]
Verified on 2.5.9-rc1
Created an rke1 node driver cluster on the ember UI and see that the etcd backup logs are moved to **DEBUG**
```
2021/06/07 15:19:32 [DEBUG] [etcd-backup] checking backups for cluster [local]
2021/06/07 15:24:32 [DEBUG] [etcd-backup] checking backups for cluster [c-wx8l7]
2021/06/07 15:14:32 [DEBUG] [etcd-backup] cluster [c-wx8l7] has no backups, creating first backup
2021/06/07 15:14:32 [DEBUG] [etcd-backup] cluster [c-wx8l7] new backup is created: c-wx8l7-rl-fj9zf
```
